src/repart/repart.c

```diff
@@ -7427,9 +7443,13 @@ static bool need_fstab(const Context *context) {
 }
 
 static int context_fstab(Context *context) {
+        const char *sd_hdr =  "# Automatically generated by systemd-repart";
+        const char *uc_hdr =  "# User provided content";
```

Comment:
please define as `const char sd_hdr[] = `, i.e. only define as array, not as ptr as array.

Reply:
so given these are const strings i think it would be better to define these as macros actually

Reply:
hmm, i'd prefer if we'd enclose the generated stuff with kind of a start and end comment, rather than have a start for the user content. After all we don't know if it's actually user content, might be something else, generated from some other tool. hence let's focus on our stuff, and say where it starts and ends and make no comment about the other stuff.

also, maybe may the strings more special, so that accidental clashes are very unlikely. For example maybe include ↓ and ↑ in the comments

Reply:
Done